cmd/podman/runlabel.go

```diff
@@ -146,10 +149,31 @@ func runlabelCmd(c *cliconfig.RunlabelValues) error {
 		return err
 	}
 	if !c.Quiet {
-		fmt.Printf("Command: %s\n", strings.Join(cmd, " "))
+		fmt.Printf("command: %s\n", strings.Join(cmd, " "))
 		if c.Display {
 			return nil
 		}
 	}
+
+	// If container already exists && --replace given -- Nuke it
+	if c.Replace {
+		for i, entry := range cmd {
+			if entry == "--name" {
+				ctr, err := runtime.LookupContainer(cmd[i+1])
```

Comment:
Does this work correctly if cmd[i+1] is an environment variable like $NAME or $IMAGE?